airflow/www/api/experimental/endpoints.py

```diff
@@ -150,23 +284,39 @@ def get_dag_code(dag_id):
         return response
 
 
+@api_experimental.route('/dags/<string:dag_id>/tasks', methods=['GET'])
+@requires_authentication
+def tasks(dag_id):
+    """Returns a JSON with all tasks associated with the dag_id. """
+    try:
+        task_list = list()
+        task_ids = get_tasks(dag_id)
+        for task_id in task_ids:
+            task_list.append(get_task_as_dict(dag_id, task_id))
+
+    except AirflowException as err:
+        _log.info(err)
+        response = jsonify(error="{}".format(err))
```

Comment:
Can simply this with just ```jsonify(error=error_message)```

Reply:
This is not possible, as these statements throw error messages stating type error, something like the following :
TypeError: Object of type 'DagNotFound' is not JSON serializable